Resources/Prototypes/Entities/Mobs/NPCs/silicon.yml

```diff
@@ -58,14 +59,17 @@
     - Stutter
     - Electrocution
   - type: NameIdentifier
-    group: GenericNumber
+    group: Silicon
+  - type: Examiner
+  - type: ContentEye
```

Comment:
this should just be inheriting from `BaseMob` rather than adding these in manually. that can be said for a lot of these components, tbh.